framework/src/mesh/GeneratedMesh.C

```diff
@@ -73,6 +73,43 @@ GeneratedMesh::GeneratedMesh(const InputParameters & parameters) :
 {
   if (_gauss_lobatto_grid && (_bias_x != 1.0 || _bias_y != 1.0 || _bias_z != 1.0))
     mooseError("Cannot apply both Gauss-Lobatto mesh grading and biasing at the same time.");
+
+  // All generated meshes are regular orthogonal meshes
+  _regular_orthogonal_mesh = true;
```

Comment:
TRIs and TETs are still valid. "regular orthogonal mesh" describes the shape of the domain, not the elements themselves. I probably should have called this "hyprerectangle" or "orthotope" instead but it's in there now :)